happy_birthder_script_tests.py

```diff
@@ -471,6 +485,8 @@ def main():
     parser.add_option('-w', '--wait', dest='wait',
                       help='allows specifying time '
                            'for waiting reminder\'s work(secs)')
+    parser.add_option('-g', '--groups', dest='exp_groups',
+                      help='allows specifying private groups expected to exist')
```

Comment:
`allows specifying the private groups which must be in Rocket.Chat when the tests are running`